### PR TITLE
Fixing python 3 syntax and add r package

### PR DESCRIPTION
**Pull request by yonghaoy**
- print statement does not work for python3.
- r upgrade is missing in GHA
- also bump RStudio version

### CODE REVIEW

File: scripts/generate_package_docs.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def get_docs(params):
 
   docs = {}
   for tool in tools:
-    print "Writing doc for tool {}".format(tool)
+    print ("Writing doc for tool {}".format(tool))
```
Comment (yonghaoy) [inline scripts/generate_package_docs.py:24]: don't know python 2 quite well, would this work in python2? (I assume Jenkins job still uses python2)

Comment (yonghaoy) [reply scripts/generate_package_docs.py:24]: @jdcanas

**Review by jdcanas — APPROVED** [review]

File: config/conf.json [hunk 2]
```diff
@@ -161,7 +161,7 @@
             "packages" : {
                 
             },
-            "version" : "3.16.0",
+            "version" : "3.16.0.1",
```
[the remark this answers is not in the document]
Comment (yonghaoy) [reply config/conf.json:164]: The issue is we are still using 3.16.0 RStudio https://github.com/DataBiosphere/terra-docker/blob/master/terra-rstudio-aou/Dockerfile#L2
Shall we: 1: disconnect bioconductor version with ours, like terra-aou-rstudio 0.0.1 refer to 3.16.0
2: Add a our patch version as suffix like this.

wdyt?

[the remark this answers is not in the document]
Comment (jdcanas) [reply config/conf.json:164]: I think we should disconnect the version, like we do for `terra-jupyter-r` with reference to `terra-jupyter-base`

[the remark this answers is not in the document]
Comment (yonghaoy) [reply config/conf.json:164]: SG, I will start from 0.1.1 then

**Pull request merged**